packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -678,11 +684,10 @@ export default function useAutocomplete(props) {
     }),
     getPopupProps: () => ({
       role: 'presentation',
-      id: `${id}-popup`,
     }),
     getListboxProps: () => ({
       role: 'listbox',
-      id: `${id}-listbox`,
+      id: `${id}-popup`,
```

Comment:
The popup is the generic term for the "list" of possible items. It can also be a grid or tree. It makes more sense if you look at the aria-controls which doesn't care if you have a listbox or grid or tree. Just that the element in question is the popup with the possible items.

Reply:
This seems correct, oops 🙃. What do you think of killing the `getPopupProps()` method and renaming `PopupComponent` to `PopperComponent`? (for future changes)

Reply:
Why Popup to Popper?

Reply:
I thought that the popup was a different element to the listbox. As you raised it out, it's not. Because the default value for PopupComponent is a Popper, the change could make the API more intuitive.

Reply:
Yeah that makes sense.